src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,33 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+        LocalDate startDate = LocalDate.parse(dateFrom, formatter);
+        LocalDate lastDate = LocalDate.parse(dateTo, formatter);
+
+        StringBuilder result = new StringBuilder("Report for period ").append(dateFrom)
+                .append(" - ").append(dateTo);
+
+        for (String name: names) {
```

Comment:
```suggestion
        for (String name : names) {
```
always use ctrl alt L for formatting before committing